.github/workflows/code_review.yml

```
@@ -0,0 +1,58 @@
+      - name: test
+        run:
+          echo "test worflow"
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # Fetch all history for all branches and tags
+
+      - name: Extract review parameters
+        id: extract
+        if: github.event_name == 'issue_comment'
+        run: |
```

actionlint (1.7.4)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)